src/cli/download_tracker.rs

```diff
@@ -81,7 +74,7 @@ impl DownloadTracker {
                 true
             }
             Notification::Install(In::Utils(Un::DownloadDataReceived(data))) => {
-                if tty::stdout_isatty() {
+                if process().is_stdout_a_tty() {
```

Comment:
We could also spell this `process().stdout().is_a_tty()`. I don't know that it makes much difference. What do you think?